WordPress/Classes/ViewRelated/Reader/WPImageViewController.m

```diff
@@ -165,13 +157,13 @@ - (void)hideBars:(BOOL)hide animated:(BOOL)animated
 
 - (void)centerImage
 {
-    CGFloat scaleWidth = _scrollView.frame.size.width / _imageView.image.size.width;
-    CGFloat scaleHeight = _scrollView.frame.size.height / _imageView.image.size.height;
+    CGFloat scaleWidth = CGRectGetWidth(self.scrollView.frame) / self.imageView.image.size.width;
+    CGFloat scaleHeight = CGRectGetHeight(self.scrollView.frame) / self.imageView.image.size.height;
```

Comment:
The denominators are taken from an image not a view frame. The image can have different resolution that the imageView.frame.